development/mimir-microservices-mode/docker-compose.jsonnet

```diff
@@ -296,7 +296,6 @@ std.manifestYamlDoc({
       command: [
         '--config.file=/etc/prometheus/prometheus.yaml',
         '--enable-feature=exemplar-storage',
-        '--enable-feature=native-histograms',
```

Comment:
Let's keep this commented out, so that people who want to test native histograms don't need to look up the correct flag.